code/modules/mob/living/simple_animal/hostile/huntersprey.dm

```diff
@@ -0,0 +1,338 @@
+/mob/living/simple_animal/hostile/renderpatriarch
+	name = "render patriarch"
+	desc = "A hulking monster out of a living nightmare. Its powerful hide can shrug off advanced lasers with ease and its heavily armored skin requires powerful ballistics to pierce."
+	icon = 'icons/mob/renderpatriarch.dmi'
+	icon_state = "rpatriarch"
+	icon_living = "rpatriarch"
+	icon_rest = "deathclaw_sleep"
+	icon_dead = "deathclaw_dead"
+	icon_gib = "deathclaw_gib"
+	aggro_vision_range = 12
+	environment_smash = 2
+	break_stuff_probability = 95
+	attacktext = "mauled"
+	faction = "deathclaw"
+	maxHealth = 2200
+	health = 2200
+	melee_damage_lower = 30
+	melee_damage_upper = 35
+	old_x = -16
+	speed = 0.8
+	var/zoomeye = 7
+	see_invisible = 15
+	var/regenrate = 25
+	var/spinecooldown = 0
+	var/roarcooldown = 0
+	var/chargecooldown = 0
+	var/charging = 0 //Are we charging right now?
+	var/fatigue = 0 //How tired are we?
+	var/maxfatigue = 6 //How tired can we become?
+	old_y = 0
+	projectiletype = /obj/item/projectile/renderspine
+	default_pixel_x = -16
+	pixel_x = -16
+	pixel_y = 0
+	meat_amount = 6
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/primal
+	can_burrow = FALSE
+	mob_size =  3  // The same as Hivemind Tyrant
+	attack_sound = 'sound/xenomorph/alien_bite2.ogg'
+
+/obj/item/projectile/renderspine
+	name = "spine"
+	icon_state = "SpearFlight"
+	damage_types = list(BRUTE = 12)
+	step_delay = 2
+
+/mob/living/simple_animal/hostile/renderpatriarch/death()
+	..()
+	visible_message("<b>[src]</b> roars loudly as it falls apart, exploding at the seams! Its last meal drops down.")
+	var chancedrop = rand(1,3)
+	if(chancedrop == 1)
+		new /obj/item/clothing/suit/armor/vest/technomancersuit (src.loc)
+		new	/obj/item/weapon/gun/energy/laser/railgun/pistol (src.loc)
+		new /obj/item/weapon/storage/belt/utility/full (src.loc)
+		qdel(src)
+		return
+	if(chancedrop == 2)
+		new /obj/item/weapon/gun/projectile/boltgun/sa (src.loc)
+		new	/obj/item/ammo_magazine/rifle_75 (src.loc)
+		new /obj/item/ammo_magazine/ammobox/rifle_75 (src.loc)
+		new /obj/item/clothing/suit/space/void/SCAF/blackshield (src.loc)
+		qdel(src)
+		return
+	if(chancedrop == 3)
+		new /obj/item/weapon/tool/sword/katana/nano (src.loc)
+		new /obj/item/clothing/suit/armor/vest/soteriasuit (src.loc)
+		new /obj/item/clothing/head/helmet/soteriasuit (src.loc)
+		qdel(src)
+		return
+	qdel(src)
+	return
+
+/mob/living/simple_animal/hostile/renderpatriarch/UnarmedAttack()
+	..()
+	if(fatigue <= maxfatigue)
+		fatigue = fatigue + 1
+	if(fatigue > 0 )
+		speed = initial(speed) * fatigue
+		melee_damage_lower = initial(melee_damage_lower) - fatigue * 2
+		melee_damage_upper = initial(melee_damage_upper) - fatigue * 2
+
+/mob/living/simple_animal/hostile/renderpatriarch/death(gibbed, deathmessage = "lets out a gutteral snarl before it crumbles to the ground dead!")
```

Comment:
Guttural.

Reply:
https://github.com/sojourn-13/sojourn-station/blob/74e60b1baf93e3e3ead7c7b2c5f9c8381895f9e7/code/modules/mob/living/simple_animal/hostile/big.dm#L65

That's one of the lines I **didn't** change. Haha! I'll take care of it.